standard/expressions.md

```diff
@@ -1402,6 +1407,58 @@ Then:
 
 *end example*
 
+### §tuple-literal-expressions-new-clause Tuple literal expressions
+
+A tuple literal consists of two or more tuple literal elements, each of which is optionally named.
```

Comment:
Is the intent to specify https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/infer-tuple-names.md separately?